### DISCUSSION TITLE
第4节课作业(14班)

### DISCUSSION THREAD

**Discussion by NewDaNew** [InternLM/Tutorial#261] [作业]
冲！！

**Comment by ZZfive** [comment]
https://sincere-papyrus-971.notion.site/b0e3ae74998f4ee690f0b0d350e16264?pvs=4

**Comment by Nobitaxi** [comment]
https://blog.csdn.net/weixin_44288092/article/details/135583040

**Comment by SingaCycle** [comment]
https://blog.csdn.net/z1370378301/article/details/135686662 第四节课作业+笔记

**Comment by xilimala** [comment]
XTuner 大模型单卡低成本微调实战 - 西里马拉的文章 - 知乎
https://zhuanlan.zhihu.com/p/677690482

**Comment by Mercuryoo** [comment]
https://t3d0h0gdxm.feishu.cn/docx/QWK7dMz0So6efPxQcxEcxWhHnF2?from=from_copylink

**Comment by shiningwhite-cmd** [comment]
https://www.yuque.com/shanxiaobai/xt4xo5/gnyp27wb03pa0wha?singleDoc# 《微调》

**Comment by lizihe11** [comment]
https://gitee.com/kiddylzh/intern-lm-tutorial-note/blob/master/task/fourth%20task.md
